cmd/nerdctl/image/image_remove_test.go

```diff
@@ -302,3 +305,58 @@ func TestRemove(t *testing.T) {
 
 	testCase.Run(t)
 }
+
+// TestIssue3016 tests https://github.com/containerd/nerdctl/issues/3016
+func TestIssue3016(t *testing.T) {
+	testCase := nerdtest.Setup()
+
+	const (
+		alpineImageName  = "alpine"
+		busyboxImageName = "busybox"
+		tagIDKey         = "tagID"
+	)
+
+	testCase.SubTests = []*test.Case{
+		{
+			Description: "Issue #3016 - Tags created using the short digest ids of container images cannot be deleted using the nerdctl rmi command.",
+			Require: test.Require(
+				test.Not(test.Windows),
+			),
+			Setup: func(data test.Data, helpers test.Helpers) {
+				helpers.Ensure("pull", alpineImageName)
+				helpers.Ensure("pull", busyboxImageName)
+
+				img := nerdtest.InspectImage(helpers, busyboxImageName)
+				tagID := strings.TrimPrefix(img.RepoDigests[0], "busybox@sha256:")[0:8]
+				assert.Equal(t, len(tagID), 8)
+
+				helpers.Ensure("tag", alpineImageName, tagID)
+
+				data.Set(tagIDKey, tagID)
+			},
+			Cleanup: func(data test.Data, helpers test.Helpers) {
+				helpers.Anyhow("rmi", alpineImageName)
```

Comment:
For cleanups, go with `"rmi", "-f"` by default.

Reply:
Thanks!!! Fix it.

Reply:
Fixed